config/ZExpELFormFactorModel.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<alg_conf>
+
+<!--
+Configuration sets for the z-expansion axial form factor model.
+
+Algorithm Configurable Parameters:
+.........................................................................................
+Name        Type    Optional   Comment                    Default
+.........................................................................................
+QEL-T0      double  Yes        z convergence parameter    GPL value: QEL-T0
+QEL-Z_A1    double  Yes        z expansion coefficient    GPL value: QEL-Z_A1
+QEL-Z_A2    double  Yes        z expansion coefficient    GPL value: QEL-Z_A2
+QEL-Z_A3    double  Yes        z expansion coefficient    GPL value: QEL-Z_A3
+QEL-Z_A4    double  Yes        z expansion coefficient    GPL value: QEL-Z_A4
+QEL-Z_A5    double  Yes        z expansion coefficient    GPL value: QEL-Z_A5
+QEL-Z_A6    double  Yes        z expansion coefficient    GPL value: QEL-Z_A6
+QEL-Z_A7    double  Yes        z expansion coefficient    GPL value: QEL-Z_A7
+QEL-Z_A8    double  Yes        z expansion coefficient    GPL value: QEL-Z_A8
+QEL-Z_A9    double  Yes        z expansion coefficient    GPL value: QEL-Z_A9
+QEL-Z_A10   double  Yes        z expansion coefficient    GPL value: QEL-Z_A10
+QEL-Kmax    int     Yes        max summation index        GPL value: QEL-Kmax
+QEL-Tcut    double  Yes        ??                         GPL value
+
+QEL-Q4limit bool    Yes        1/Q^4 limit conditions     GPL value: QEL-Q4limit
+
+-->
+
+  <param_set name="Default">
+
+    <param type="string"   name="CommonParam"> QuasiElastic  </param>
+
+    <param type="bool"   name="QEL-Q4limit"> true   </param>
+    <param type="int"    name="QEL-Kmax">    4      </param>
+
+    <param type="double" name="QEL-T0">     -0.21   </param>
+    <param type="double" name="QEL-Tcut">    0.0779198 </param> <!-- 4*m_pi^2 -->
+    <param type="double" name="QEL-Z_AP1"> -1.48596      </param>
+    <param type="double" name="QEL-Z_AP2"> -0.0961045    </param>
+    <param type="double" name="QEL-Z_AP3"> 1.82337       </param>
+    <param type="double" name="QEL-Z_AP4"> 1.28705       </param>
+    <param type="double" name="QEL-Z_BP1"> -4.0879       </param>
+    <param type="double" name="QEL-Z_BP2"> 0.175516      </param>
+    <param type="double" name="QEL-Z_BP3"> 4.86186       </param>
+    <param type="double" name="QEL-Z_BP4"> -0.982798     </param>
+    <param type="double" name="QEL-Z_AN1"> 0.0842527     </param>
+    <param type="double" name="QEL-Z_AN2"> -0.278572     </param>
+    <param type="double" name="QEL-Z_AN3"> -0.153916     </param>
+    <param type="double" name="QEL-Z_AN4"> 0.354         </param>
+    <param type="double" name="QEL-Z_BN1"> 2.70678       </param>
+    <param type="double" name="QEL-Z_BN2"> -0.424305     </param>
+    <param type="double" name="QEL-Z_BN3"> -2.66245      </param>
+    <param type="double" name="QEL-Z_BN4"> 0.0675831     </param>
```

Comment:
This configuration will change according to something like [this](https://github.com/GENIE-MC/Generator/blob/ecc6884b81eec3ef3d909a09150aa856805c3bf5/config/BaryonResonanceDecayer.xml#L84)